### ISSUE TITLE
Export markdown may not be correct

### ISSUE THREAD

**Issue by liuyuqiangchn** [stevencohn/OneMore#978]
Hello, thank you for developing such a great software, I have encountered a problem

When exporting markdown files, paragraphs are interpreted as line breaks，For example
Version: 5.8.2x64

![ONENOTE_oIRxwXLq2m](https://user-images.githubusercontent.com/22015183/225553636-e811e63a-f3e4-49c4-8e5b-0288fed3b448.png)

**Comment by stevencohn** [comment]
Do you mean Export as HTML? BR elements are inserted when you have a _soft line break_ which can be inserted when you press Shift+Enter instead of Enter. This is correct.

If you export this as Markdown, the result should look something like the following, without HTML elements

```markdown
# Page Title Here
P1: Years may wrinkle the skin ...
P2 : When your aerials are down ... 
```

**Comment by liuyuqiangchn** [comment]
Thank you very much for your reply, maybe I didn't explain it clearly
Since P1 and P2 are two paragraphs, I want the exported markdown to show 2 paragraphs

![ONENOTE_Q5fnqRgN2t](https://user-images.githubusercontent.com/22015183/225629282-9ff0c93c-9f12-4059-bd63-728f5f50e482.png)

**Comment by stevencohn** [comment]
I'd argue that the other add-in is not correct. Your example does not show an empty line between the two paragraphs so OneMore doesn't add one.

Change it to
<img width="194" alt="image" src="https://user-images.githubusercontent.com/9964757/225631785-028b6086-4083-4afa-bfab-aaab4c77cbf2.png">

and the resultant markdown will look like this:

```
P1: Year may wrinkle the skin  

P2 : When your aerials are down  
```

**Comment by stevencohn** [comment]
I'll take a look

**Comment by liuyuqiangchn** [comment]
Thank you very much, I love this software

**Comment by stevencohn** [comment]
Will fix this under #802
